src/sock_ctrl_msg.rs

```diff
@@ -0,0 +1,494 @@
+// Copyright 2017 The Chromium OS Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
```

Comment:
Do we need change "LICENSE" to the real license file name?

Reply:
yes, I think so